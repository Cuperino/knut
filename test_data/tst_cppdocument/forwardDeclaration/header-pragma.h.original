#pragma once

namespace Foo {

class Something
{
};

} // namespace Foo
