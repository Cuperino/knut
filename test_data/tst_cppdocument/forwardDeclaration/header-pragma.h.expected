#pragma once

namespace SFoo {
namespace SBar {
struct Uno;
}
}

struct Neo;

namespace Foo {
namespace Bar {
class FooBar;
}
}

class Foo;

namespace Foo {

class Something
{
};

} // namespace Foo
